servers/src/main/java/tachyon/master/block/BlockMasterWorkerServiceHandler.java

```diff
@@ -31,6 +32,7 @@
 /**
  * This class is a Thrift handler for block master RPCs invoked by a Tachyon worker.
  */
+@ThreadSafe
```

Comment:
Isn't the fact that `BlockMaster` is not thread safe really bad? That would imply that concurrent requests to `BlockMaster` can race.